introduction.md

```diff
@@ -1,3 +1,7 @@
 introduction
 I am peter.
 my favorite code school path is JavaScript
+
+* Git
+* Javascript
+* Java
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?